java/arcs/core/host/EntityHandleManager.kt

```diff
@@ -40,11 +40,13 @@ typealias CollectionSenderCallbackAdapter<E> =
  * [ReadableSingleton] for [HandleMode.Read]. To obtain a [HandleHolder], use
  * `arcs_kt_schema` on a manifest file to generate a `{ParticleName}Handles' class, and
  * invoke its default constructor, or obtain it from the [BaseParticle.handles] field.
+ *
+ * TODO(cromwellian): Add support for creating Singleton/Set handles of [Reference]s.
```

Comment:
Ray, I'm not 100% sure what should be done (if anything, yet?) regarding creating/supplying SingletonHandle<Reference> and CollectionHandle<Reference> instances for PA.

Reply:
Let me think about this a bit and we'll do it in a follow on PR, because it will likely involve changes to schema2kotlin, and the generated Entities/EntitySpec classes.  I wouldn't block on this.